corehq/apps/users/models.py

```diff
@@ -2548,7 +2550,3 @@ def get(self, user_id):
             user = CouchUser.get_by_user_id(user_id)
             self.cache[user_id] = user
             return user
-
-
-from .signals import *
```

Comment:
See [Where should this code live?](https://docs.djangoproject.com/en/1.8/topics/signals/#connecting-receiver-functions)